### DISCUSSION TITLE
how can add  konnect paid and enterprise kong plugin into kong manager?

### DISCUSSION THREAD

**Discussion by ChainCoderX** [Kong/kong-manager#160] [General]
I want to use OAuth 2.0 Introspection and Kafka Upstream plugins on the kong-manager.
but it  is   konnect paid and enterprise.

**Comment by Leopoldthecoder** [comment]
Hello @wade-liwei, Konnect paid plugins and Kong Enterprise plugins can be accessed from Konnect and Kong Gateway Enterprise. This repo is for Kong Gateway OSS only.
